build.boot

```diff
@@ -91,16 +86,20 @@
     (serve :dir "target")
     (repl :server true)))
 
-(deftask docs
-  []
-  (comp
-    (build-docs)))
-
 (deftask testing
   []
   (merge-env! :source-paths #{"src/test"})
   identity)
 
+(deftask docs
+  []
+  (comp
+   (codox :name "workflo/macros"
+          :source-paths #{"src/main"}
+          :output-path "api-docs"
```

Comment:
Could do, the problem is that's where our more tutorial-like Markdown documentation for the different macros resides. We could move that somewhere else of course.